Samples/WinMLSamplesGallery/WinMLSamplesGallery (Package)/WinMLSamplesGallery (Package).wapproj

```diff
@@ -64,5 +64,13 @@
     </ProjectReference>
   </ItemGroup>
   <Import Project="$(WapProjPath)\Microsoft.DesktopBridge.targets" />
-  <Import Project="$(AppxTargetsLocation)Microsoft.WinUI.AppX.targets" />
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AI.MachineLearning" Version="1.9.0-dev-20210916-0238-83dc22585" />

```

Comment:
This is referencing an unreleased version of the NuGet. Please reference 1.9.0 when it is released in a couple days.

Reply:
We had to change it to reference 1.9 to fix a bug that was causing the image classifier sample to crash due to custom tensorization models. I think @smk2007 can explain more

Reply:
1.9 has been released, can you update the reference?